### PR TITLE
Adds additional variants of 10mm ammunition.

### PR DESCRIPTION
**Pull request by theColdflame**
Improved version of #5569

Adds three new types of 10mm ammunition, and the relevant magazines. Does not make the ammunition available to players.

AP- Ignores the majority of the target's armor, but does reduced damage. Buffed since its previous iteration.
HP- Does significantly increased base damage, but heavily increases the target's armor effectiveness. This one might not be too well placed in the hands of traitors.
Incendiary- What it says on the tin. Ignites the target, though the fire can easily be extinguished.

Rebalanced silencer rounds may return alongside other variants in a later PR, but this is a more reasonable starting point for 10mm variants.
:cl: Coldflame
rscadd: Adds three new variants of 10mm ammunition, AP, HP, and Incendiary. Currently admin-only.
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
`armour_penetration` isn't a percentage. It's a flat amount.

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -112,6 +112,20 @@
 /obj/item/projectile/bullet/midbullet3
 	damage = 30
 
+/obj/item/projectile/bullet/midbullet3/hp
+	damage = 45
+	armour_penetration = -100
+
+/obj/item/projectile/bullet/midbullet3/ap
+	damage = 27
+	armour_penetration = 80
```
Comment (Fox-McCloud) [inline code/modules/projectiles/projectile/bullets.dm]: Ignoring 80 armor is awfully high; this is going to fully negate even DS armor.

**Comment by theColdflame** [comment]
Unarmored targets appear to take full damage from hollow points (perhaps since their armor isn't calced at all). I've adjusted it to -70, which means it'll be still have SOME effectiveness against moderately armored targets (still borderline useless, ~7 damage against an officer armor and stopped dead by bullet vests or the hoscoat). 

AP's been nerfed to 40 penetration, meaning it'll ignore moderate armor but still face issues with specialized protection or admin/event gear.

File: code/modules/projectiles/projectile/bullets.dm [hunk 2]
```diff
@@ -112,6 +112,20 @@
 /obj/item/projectile/bullet/midbullet3
 	damage = 30
 
+/obj/item/projectile/bullet/midbullet3/hp
+	damage = 45
+	armour_penetration = -70
```
Comment (Fox-McCloud) [inline code/modules/projectiles/projectile/bullets.dm]: I'd personally recommend a value of something like 40 and -40 that way these aren't 100% worthless against someone with a teensy amount of armor, but aren't super dead kill against those without it.

**Pull request merged**